src/qml/WelcomeScreen.qml

```diff
@@ -403,6 +403,7 @@ Page {
         }
         QfButton {
           id: localProjectButton
+          visible: Qt.platform.os !== "ios" // iOS does not allow random file access. Disable local data picker until we have a proper import/export workflow like on Android
```

Comment:
I think it's better to disable here than hide, for a few reasons: i) hidden could lead to users misunderstanding we are not planning to support non-cloud projects the way we do on android, and ii) reduces the chance of someone supporting the effort of proper implementation :)

```suggestion
          enabled: Qt.platform.os !== "ios" // iOS does not allow random file access. Disable local data picker until we have a proper import/export workflow like on Android
```

Reply:
But: Hidden will also make people think "what did I do wrong"

Reply:
@m-kuhn , how though, if someone isn't familiar with QField on Android, he/she/they will not know there's a potential local dataset support no?

Also, `enabled: false` is what we did to tease the cloud side of things in the QField 1.x cycle.

Reply:
Disabling the button will help them understand it's not available right now but with now hint what they can do to fix the problem which leaves it as "broken functionality cluttering my screen and the easiest thing I can do is giving a bad review that functionality is borken"?